### DISCUSSION TITLE
Migrating users to newer version of Nextcloud

### DISCUSSION THREAD

**Discussion by rspierenburg** [nextcloud/user_migration#687] [Q&A]
I'm preparing to migrate from an Odroid HC1 running NextcloudPi (Nextcloud v25.0.13) to a new pc running Nextcloud AIO (Nextcloud v28).
I was wondering if there would be any issues using this tool to migrate my users due to the version difference?
Thanks,
Rob

**Comment by Pytal** [comment]
There should not be any conflicts during migration as migrators are versioned but you may open an issue if you run into anything
